dotCMS/src/main/java/com/dotmarketing/util/StringUtils.java

```diff
@@ -441,4 +441,37 @@ public static boolean hasWhiteSpaces(final String in){
       return pattern.matcher(in).find();
    }
 
+    /**
+     * Method to convert camel case convention into snake case
+     * @param parameter
+     * @return
+     */
+    public static String convertCamelToSnake(String parameter)
+    {
+        String result = "";
+
+        // Append first character(in lower case)
+        char c = parameter.charAt(0);

```

Comment:
rename to first letter or meaningful var name

Reply:
This feedback has been addressed